packages/nextra-theme-docs/src/components/sidebar.tsx

```diff
@@ -358,33 +357,18 @@ export function Sidebar({
             </div>
           </div>
 
-          {!hasMenu ? null : (
+          {hasMenu && (
             <div className="nextra-sidebar-menu mx-4 border-t shadow-[0_-12px_16px_white] dark:border-neutral-800 dark:shadow-[0_-12px_16px_#111]">
-              <div className="flex gap-1 bg-white py-4 pb-4 dark:bg-dark">
+              <div className="flex gap-1 bg-white py-4 pb-4 dark:bg-dark justify-between">
                 {config.i18n ? (
-                  <div className="relative flex-1">
+                  <div className="relative">
```

Comment:
fixes button styles, locale button we'll no longer take all place
<img width="292" alt="image" src="https://user-images.githubusercontent.com/7361780/184453135-39009c80-90e3-4a6c-84d1-fa9c6bc55a49.png">